treeherder/webapp/api/push.py

```diff
@@ -71,6 +75,14 @@ def list(self, request, project):
 
             pushes = pushes.filter(repository=repository)
 
+        search_param = filter_params.get("search")
+        if search_param:       
+            pushes = pushes.annotate(
+                search=SearchVector(
+                    "revision", "author", config="english"
+                ) + SearchVector( "commits__revision","commits__author","commits__comments", config="english")
```

Comment:
Please remove the whitespace after the starting parenthesis and have one after each comma.